tests/test_htmIndex.py

```diff
@@ -70,26 +72,35 @@ def make_sky_catalog(out_path, size=1000):
         is_variable = np.random.randint(2, size=size)
         extra_col1 = np.random.normal(size=size)
         extra_col2 = np.random.normal(1000., 100., size=size)
+        pm_ra = -np.ones(size)*cls.proper_motion  # arcsec/year
+        pm_dec = np.zeros(size)  # arcsec/year
+        pm_ra_err = np.ones(size)*0.001  # arcsec/year
+        pm_dec_err = np.ones(size)*0.001  # arcsec/year
```

Comment:
More "I wish we could use astropy.Quantity", but I don't know that there's a feasible way to do so.